### ISSUE TITLE
What is this app about?

### ISSUE THREAD

**Issue by atulya2109** [Ritiek/rustbook-android#1]
Any hints

**Comment by Ritiek** [comment]
I am trying to make an android app to fetch data from https://doc.rust-lang.org/book/ so people can easily access the Rust documentation on a mobile. Hopefully, some kind of offline version too if it goes well. Would love a hand if you got some time!

**Comment by atulya2109** [comment]
Sure...

**Comment by atulya2109** [comment]
So, 
1.We can scrape the website.
2.Hard-code all the info.

**Comment by atulya2109** [comment]
Going with the former one the best thing that can be done is scraper+crawler on a server. The app can query the server and can get data in JSON format.
server part-python
You can set the server to look for changes every 'X' hours(probably 24) and update the content in its database.
App can query server on each restart and if there is a change update content in its local database.

**Comment by atulya2109** [comment]
If you want to keep it easy just use a webview.
That would be pretty quick and simple & there is no fun in that and you can't go offline.
Webscraper+crawler would be a little time consuming on android.
Getting html and scraping 19 to 20 webpages will slow down the app.

**Comment by atulya2109** [comment]
It would be great if I could commit directly to repo

**Comment by Ritiek** [comment]
Yeah, sure. Let's just stick with webview initially. We could work on offline usage later when the app is 
 functional and stable.

**Comment by atulya2109** [comment]
The webview drawer is not working...

**Comment by Ritiek** [comment]
Do you have a workaround? Maybe we could add the web drawer in the application drawer itself?

Sorry, I am a total idiot in Android Dev. This is more like a learning project for me.. :confused:

**Comment by atulya2109** [comment]
Yeah, that is what we are going to do.
This is too simple for a learning project as there are no clicklisteners, array adapters buttons.
Check out udacity for android course

**Comment by atulya2109** [comment]
That would be a very long drawer.
I suggest an index activity
